bridge-fungible-token-abi/src/bridge_fungible_token_abi.sw

```diff
@@ -0,0 +1,38 @@
+library bridge_fungible_token_abi;
+
+use std::{contract_id::ContractId, identity::Identity, vm::evm::evm_address::EvmAddress};
+
+abi BridgeFungibleToken {
+    /// Claim a refund for an EvmAddress if one has been registered.
+    ///
+    /// # Arguments
+    ///
+    /// * `originator` - the EvmAddress that is entitled to a refund
```

Comment:
It says `EvmAddress`, but the param is actually `b256`. I think `b256` might be the better option